src/core/Details/Details.md

```diff
@@ -0,0 +1,70 @@
+Examples:
+
+- [Basic use](./#/Components/Details?id=basic-use)
+- [Varied content](./#/Components/Details?id=varied-content)
+
+<div style="margin-bottom: 40px">
+  [Props & methods](./#/Components/Toast?id=props--methods)
+</div>
+
+### Basic use
```

Comment:
There could be a line here about the required prop of the component. 

"Provide a `summaryLabel` for the details heading" or something like that